verification_site/src/main/java/com/generalbytes/verification/service/ApplicantService.java

```diff
@@ -0,0 +1,35 @@
+package com.generalbytes.verification.service;
+
+import com.generalbytes.verification.controller.RegisterNewApplicantReq;
+import com.generalbytes.verification.data.Applicant;
+import org.springframework.stereotype.Service;
+
+import java.time.Instant;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static java.time.temporal.ChronoUnit.SECONDS;
+
+@Service
+public class ApplicantService {
+
+    private static final Long TIMEOUT_DURATION_SECONDS = 90 * 60L; // every SDK token is valid only for 90 minutes
+    private final ConcurrentHashMap<String, Applicant> CACHE = new ConcurrentHashMap<>();
```

Comment:
Use CacheBuilder.

Example is here: https://github.com/GENERALBYTESCOM/batm_public/blob/8f76ef5697f1a5887d98c88e324607b4e9a1270e/server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/XChangeExchange.java#L69